<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QWidget" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="GLWidget" name="widget" native="true">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{
background-image: url(:/new/pics/pics/back.png);
border: none;
}
#pushButton:hover{
 border-width:0.5px;	
border-style:solid;
	border-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QStackedWidget" name="popUpMenu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>201</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="save_btn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#save_btn{
background-image: url(:/new/pics/pics/save.png);
border: none;
}
#save_btn:hover{
 border-width:0.5px;	
border-style:solid;
	border-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="loadButn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#loadButn{
background-image: url(:/new/pics/pics/load.png);
border: none;
}
#loadButn:hover{
 border-width:0.5px;	
border-style:solid;
	border-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Dialog" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>341</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_2{
background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#play_btn{
background-image: url(:/new/pics/pics/play.png);
border: none;
}



#play_btn:hover{
 border-width:0.5px;	
border-style:solid;
	border-color: rgb(255, 255, 255);
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="restart_btn">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>50</width>
        <height>49</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#restart_btn{
background-image: url(:/new/pics/pics/restart.png);
border: none;
}
#restart_btn:hover{
 border-width:0.5px;	
border-style:solid;
	border-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Dialog</class>
   <extends>QWidget</extends>
   <header location="global">dialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
